assets/BrowseTables.css

```diff
@@ -11,13 +11,17 @@ body {
 
 table {
     border-collapse: collapse;
+    margin: 0 1em 0 0; /* push scrollbar to the left a bit */
 }
 
 /* table header */
 
 table thead th {
     color: #eee;
     background-color: #4682b4;
+    padding-left: 0.4ex; /* th & td consistency */
+    padding-right: 0.4ex;
+    width:2em;
```

Comment:
As I said in the discourse thread, I am now willing to sacrifice compactness. Also, I think the alignment is still not OK.